Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/Dashboard/Driver.hs

```diff
@@ -988,3 +995,107 @@ updateByPhoneNumber merchantShortId phoneNumber req = do
       AV.create aadhaarEntity
   QDriverInfo.updateAadhaarVerifiedState (cast driver.id) True
   pure Success
+
+fleetUnlinkVehicle :: ShortId DM.Merchant -> Text -> Text -> Text -> Flow APISuccess
+fleetUnlinkVehicle merchantShortId fleetOwnerId vehicleNo newDriverMobileNo = do
+  merchant <- findMerchantByShortId merchantShortId
+  vehicle <- QVehicle.findByRegistrationNo vehicleNo >>= fromMaybeM (VehicleDoesNotExist vehicleNo)
+  unless (isJust vehicle.fleetOwnerId && vehicle.fleetOwnerId == Just fleetOwnerId) $ throwError (FleetOwnerVehicleMismatchError fleetOwnerId)
+  let oldDriverId = vehicle.driverId
+  mobileNumberHash <- getDbHash newDriverMobileNo
+  newDriver <- QPerson.findByMobileNumberAndMerchant mobileIndianCode mobileNumberHash merchant.id >>= fromMaybeM (PersonDoesNotExist newDriverMobileNo)
```

Comment:
instead of mobileIndianCode it should be countrycode

Reply:
Added Mobile Country code for add fleet vehicle and unlink fleet vehicle apis